frontend/src/common/enums/notification/notification-messages/success-notification-message.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum SuccessNotificationMessage {
+  GROUP_CREATE = 'Group has been succeffully created!',
```

Comment:
```suggestion
enum NotificationMessage {
  GROUP_CREATE = 'Group has been succeffully created!',
```
don't forget about file name